Sources/String+Sweetness.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+public extension String {
+    public var isValidEmail: Bool {
+        let emailRegEx = ".*@.*"
```

Comment:
@3lvis overkill. No one is trying to intentionally register broken invalid emails, and if they are, so what? 🤔